static/js/components/PersonalTab.js

```diff
@@ -23,15 +39,74 @@ class PersonalTab extends React.Component {
     ui:             UIState,
     nextStep:       () => void,
     prevStep:       () => void,
+    dispatch:       Dispatch,
+    dashboard:      DashboardState,
+    enrollments: ProgramEnrollmentsState,
+    currentProgramEnrollment:    ProgramEnrollment,
   };
 
+  programListing = (programs: Program) => {
+    const makeMenuItems = R.map(program => (
+      <MenuItem value={program.id} key={program.id} primaryText={program.title} />
+    ));
+    const sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
+    const menuItems = R.compose(makeMenuItems, sortPrograms);
+    return menuItems(programs || {}) ;
```

Comment:
Since `programs` comes from the redux store and its initial state is an array, you should not need the `|| {}` here. And `|| {}` is also wrong because it should be an array